spec/views/users/index.html.erb_spec.rb

```diff
@@ -20,13 +21,17 @@
       end
     end
 
-    it 'has an + button to send contact request' do
+    it 'has a + button to send contact request' do
       expect(rendered).to have_button('+', count: users.length)
     end
 
     it 'does have a + button for not requested user' do
       expect(rendered).to have_css("form.button_to[action=\"/users/#{users.second.id}/contact_requests\"]")
     end
+
+    it 'is not possible to see myself in the list of all users' do
+      expect(rendered).not_to have_text(users.first.email, count: users.length)
+    end
```

Comment:
That should be also the signed_in_user instead of users.first, shouln't it?

Reply:
I think there are also some other places in this file where it has to be changed